roles/cockpit-ui/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Install cockpit-ui
+  action: "{{ ansible_pkg_mgr }} name={{ item }} state=present"
+  when: not openshift.common.is_containerized | bool
```

Comment:
Needs either with_items: list or just change {{ item }} to the name of the package. Also, maybe we can use the existing cockpit role?